src/main/java/com/lambdaworks/redis/output/NestedMultiOutput.java

```diff
@@ -40,7 +40,8 @@ public void set(ByteBuffer bytes) {
 
     @Override
     public void setError(ByteBuffer error) {
-        output.add(new RedisCommandExecutionException(decodeAscii(error)));
+        super.setError(error);
+        output.add(new RedisCommandExecutionException(this.error));
```

Comment:
I also think that adding errors to the output is an error. I'd propose to remove the method override fall back to the default error propagation.